birdman/model_base.py

```diff
@@ -1,15 +1,17 @@
-from typing import List, Sequence
+from typing import List, Sequence, Union
 import warnings
 
 import arviz as az
 import biom
 from cmdstanpy import CmdStanModel, CmdStanMCMC
 import dask
 import dask_jobqueue
+import numpy as np
 import pandas as pd
 from patsy import dmatrix
 
-from .model_util import single_fit_to_inference, multiple_fits_to_inference
+from .model_util import (single_fit_to_inference, multiple_fits_to_inference,
+                         _single_feature_to_inf)
```

Comment:
If you like the previous suggestion, you'll want to change this to 
```suggestion
                         _single_feature_to_inf, concatenate_inferences)
```